cloud/docker/docker.py

```diff
@@ -1347,9 +1402,11 @@ def main():
             memory_limit    = dict(default=0),
             memory_swap     = dict(default=0),
             docker_url      = dict(),
-            docker_tls_cert = dict(),
-            docker_tls_key  = dict(),
-            docker_tls_cacert = dict(),
+            use_tls         = dict(default=None, choices=['no', 'encrypt', 'verify']),
+            tls_client_cert = dict(required=False, default=None, type='str'),
```

Comment:
I'd been assuming that `required=False, default=None, type='str'` were the defaults. Any reason not to just use `dict()` instead?